src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -599,11 +599,6 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
         return true;
       }
     }
-    if (this.hearingRequestMainModel.hearingDetails.duration){
-      if (HearingsUtils.hasHearingDurationChanged(this.hearingRequestMainModel.hearingDetails.duration, this.serviceHearingValuesModel.duration)){
-        return true;
-      }
-    }
```

Comment:
there seems to be a path through this function where it drops of the end without an explicit return, which is bad. Removing this block would seem to erxpand the chance of this happening. I'd suggest adding return false as the last line of the function.

Reply:
I think the return false already exists on line 607, is this what you meant?

Reply:
sorry, yes